File: mario.c

#include <stdio.h>
#include <cs50.h>

int main(void)
{
  int Height, CountH, CountW, Width, Left;
  Height = get_int("Hoe hoog wordt de pyramide? ");
  while (Height < 1 || Height > 8 ){Height = get_int("Kies een hoogte tussen 1 en 8: ");}

  for (CountH = 0;CountH < Height ; CountH++)
  {
    Left = Height-CountH; 

    for (CountW = 1; CountW < Left; CountW++)
    {
	printf(" ");
    }
    for (CountW = 0; CountW <= CountH; CountW++)
    {
	printf("#");
    }
    printf("\n");
  }

}

File: python/mario.py
from cs50 import get_int

height = get_int("Height: ")

while height < 0 or height > 23:
	height = get_int("Height: ")

if height == 0:
	print()
else:
	count = 0
	width = 0
	rtrn = ""
	i = height - count + 1

	for count in range (height):
		for i in range (height-width-1):
			rtrn = rtrn + " "
			i += 1
		for width in range(count):
			rtrn = rtrn + "#"
			width += 1
		#if ( i != height):
		print (rtrn + "##")
		i = height - count + 1
		rtrn = ""
		count += 1
		width += 1
